### PR TITLE
Fix/3547  avoid handling tlm twice in the frontend

### PR DESCRIPTION
**Pull request by raphj**
#3547

:warning: this PR probably fixes the issues, but some discussions in the ticket are unresolved, so keep this in mind when reviewing this code.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/util/LiveMessageManager.js [hunk 1]
```diff
@@ -61,6 +62,13 @@ export class LiveMessageManager {
       this.broadcastChannel.postMessage({ status: this.status })
     }
 
+    if (this.eventSource && (message.status || message.tlm)) {
+      // There are two leaders.
+      if (message.fromTabId > this.tabId) {
```
Comment (lebouquetin) [inline frontend/src/util/LiveMessageManager.js:67]: Is this related to the master election? Is the strategy described somewhere (centralised) in the code? If not it would be nice to describe the "algorithm".

For example, here: you have two leaders. Ok. There is a message coming from a tabId "greater" than the current tabId. Why do this lead to a closeConnection()?

**Review by grignards — APPROVED** [review]
